README.md

```diff
@@ -45,9 +47,10 @@ We expect members of the community participating in conversations here to act in
 - Use logical fallacies in arguments
 - Recreate an issue you feel was unjustly closed -- if you feel an issue was unjustly closed, feel free to add a comment to the issue explaining why
 - Add comments such as 👍, +1, or "I like this idea" -- use [reactions](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)! We can measure those :)
+- Request a feature/change to the bot. It has its own repo which can be found [here](https://github.com/Minehut/MinehutBOT/issues).
 
 ## Submitting An Issue
 > ⚠️ **Important!** If you discover a potential security exploit or vulnerability, please do not report it here! Check our [Security Policy](https://github.com/Minehut/Meta/security/policy) for more information.
 
-- Once you're ready to submit your bug report or feature request, head over to [this page](https://github.com/Minehut/Meta/issues/new/choose) to begin the process
-- You can also comment on other issues and participate in other conversations. Remember, if you want to show support or general disagreement of an issue, use 👍 and 👎 reactions. You can also [sort issues by 👍 reactions](https://github.com/Minehut/Meta/issues?q=is%3Aissue+sort%3Areactions-%2B1-desc) (also [by 👎 if you'd like](https://github.com/Minehut/Meta/issues?q=is%3Aissue+sort%3Areactions--1-desc))
+- Once you're ready to submit your bug report or feature request, head over to [this page](https://github.com/Minehut/Meta/issues/new/choose) to begin the process.
+- You can also comment on other issues and participate in other conversations. Remember, if you want to show support or general disagreement of an issue, use 👍 , and 👎 reactions. You can also [sort issues by 👍 reactions](https://github.com/Minehut/Meta/issues?q=is%3Aissue+sort%3Areactions-%2B1-desc) (also [by 👎 if you'd like](https://github.com/Minehut/Meta/issues?q=is%3Aissue+sort%3Areactions--1-desc))
```

Comment:
Following US rules again, there's really no need to change this.

Reply:
As I said above, I'm not contested to this change.

Reply:
As I said above, I'm not contested to this change